scripts/ec2cleanup.py

```diff
@@ -164,10 +166,34 @@ def delete_stacks(provider_mgmt, excluded_stacks, stack_template, output):
         logger.exception('Exception in %r', delete_stacks.__name__)
 
 
+def delete_snapshots(provider_mgmt, excluded_snapshots, output):
+    snapshot_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for snapshot in provider_mgmt.list_own_snapshots():
+            snapshot_id = snapshot.get("SnapshotId")
+            if excluded_snapshots and snapshot_id in excluded_snapshots:
+                logger.info(" Excluding Snapshot with id: %r", snapshot_id)
+                continue
+            else:
+                if provider_mgmt.delete_snapshot(snapshot_id=snapshot_id):
```

Comment:
Are there any snapshot ID's we'll want to exclude by default? this appears to delete all snapshots owned by 'self' on ec2

Reply:
I don't think there are any snapshots that we will want to exclude. I think we should rather make AMIs from snapshots that we will want to use in the future, but I added this exclude option because it's used in all other methods so I wanted to make it unified.